frontend/e2e/cypress.config.js

```diff
@@ -0,0 +1,31 @@
+const { defineConfig } = require('cypress');
+
+// load the environment variables from the local .env file
+require('dotenv').config();
+// Just for debugging missing env vars
+// console.log(process.env)
+
+module.exports = defineConfig({
+  viewportWidth: 1000,
+  viewportHeight: 700,
+  chromeWebSecurity: false,
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
+    setupNodeEvents(on, config) {
+      require('./cypress/plugins/index.js')(on, config);
+
+      config.env = config.env || {};
```

Comment:
This should probably be `??` rather than `||`

Reply:
probably something to add to our coding guidelines